source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -544,14 +546,26 @@ class WordDocument(UIADocumentWithTableNavigation,WordDocumentNode,WordDocumentB
 	def event_textChange(self):
 		# Ensure Braille is updated when text changes,
 		# As Microsoft Word does not fire caret events when typing text, even though the caret does move.
-		braille.handler.handleCaretMove(self)
+		# Update braille also when tethered to review, and review position
+		# if review follows caret.
+		if not eventHandler.isPendingEvents("caret, self"):

```

Comment:
This looks incorrect, and probably should be:
```suggestion
		if not eventHandler.isPendingEvents("caret", self):
```